package/doc/sphinx/source/documentation_pages/coordinates/pickle_universe.rst

```diff
@@ -0,0 +1,117 @@
+.. Contains the formatted docstrings for the serialization of universe located 
+.. mainly in 'MDAnalysis/libs/pickle_file_io.py'
+.. _serialization:
+
+*********************************************************
+Serialization of Universes
+*********************************************************
+
+As we approach the exascale barrier, researchers are handling increasingly 
+large volumes of molecular dynamics (MD) data. Whilst MDAnalysis is a flexible
+and relatively fast framework for complex analysis tasks in MD simulations, 
+implementing a parallel computing framework would play a pivotal role in 
+accelerating the time to solution for such large datasets.
+
+To achieve a flawless implementation of parallelism, this document illustrates
+the basic idea of how current :class:`Universe` is being serialized. And what people
+should do to serialize a new reader.
+
+:class:`Universe` is serialized by serializing each necessary compartment---
+:attr:`_topology`, :attr:`_trajectory`, :attr:`anchor_name`; a new :class:`Universe` is
+reconstructed with these elements. 
+
+To make sure every Trajectory reader can be successfully
+serialized, we implement picklable I/O classes (see :ref:`implemented-fileio`).
+When the file is pickled, filename and other necessary attributes of the open 
+file handle are saved. On unpickling, the file is opened by filename.
+This means that for a successful unpickle, the original file still has to
+be accessible with its filename. To retain the current frame of the trajectory,
+:func:`_read_frame(previous frame)` will be called during unpickling.
+
+Here is an example of how a :class:`Universe` is (un)pickled:
+
+.. code-block:: python
+
+    u = MDAnalysis.Universe(topology, trajectory)
+    u.trajectory[5]
+    u_pickled = pickle.loads(pickle.dumps(u)
+    u_pickled.ts.frame == u.trajectory.ts.frame
+
+.. _simple_parallel_analysis:
+
+Simple analysis case
```

Comment:
The example together with more explanations (as asked for by @kain88-de below) would be good in a user guide document.

Reply:
What do you think about an overview (important need-to-know, basic intro) + detailed tutorial for the user guide, @yuxuanzhuang? Then you can also have separate documents on serialisation, and parallel analysis, and just link them to make it clear they are related.

Reply:
I would put the user-portion of your parallel analysis docs in the **analysis** section of the User Guide – you can start a section _Parallel Analysis_ with your general introduction and then your example how to use multiprocessing.  You can add a See Also section for PMDA for the time being. Later, sections on dask, MPI, and whatnot can be added.

Reply:
Sounds good to me.